lib/internal/Magento/Framework/Code/Test/Unit/GeneratorTest.php

```diff
@@ -311,7 +324,7 @@ public function testGenerateClassWithExistName($fileExists): void
 
         $this->assertSame(
             Generator::GENERATION_SKIP,
-            $this->model->generateClass(GeneratedClassFactory::class)
+            $this->model->generateClass('Magento\GeneratedClass\Factory')
```

Comment:
Why was it replaced with FQCN?

Reply:
Hello @ihor-sviziev ! Thank for your review. This changes needed to Static tests. The tests get an error with FQCN because Factory doesn't exist.

Reply:
Please add comment there